client_options.go

```diff
@@ -241,3 +242,9 @@ func WithDisableIPV6() HttpClientOption {
 		config.disableIPV6 = true
 	}
 }
+
+func WithBandwidthTracker(enabled bool) HttpClientOption {
```

Comment:
This option should not have a parameter in my opinion. If someone wants to enable the Tracker he can add the option (which results in `config.enabledBandwidthTracker = true`) and if you do not want the tracker ommit the option completely.

Similar to the `WithDisableIPV6` option.